Add Jeffrey to Contributors list

jeffreytram opened the pull request.

jeffreytram committed:
Add Jeffrey to Contributors list

Roshanjossey commented:
Hi  @jeffreytram, I'm quite elated about your pull request. I wanna evolve this project to addresses various problems faced by first-time contributors. I'd love to learn about your journey in open source community, the problems, pain points you had etc.
Could you explain how you felt when you went through the tutorial, made a pull request and learned that I merged it?

 We’ve recently added social share  to our web app. Could you please go to https://roshanjossey.github.io/first-contributions/#social-share and share your first contribution to open source?  Also, check out projects with easy issues while you’re there.

jeffreytram commented:
Sure thing! I'd love to give some feedback!

_**First Contributions Intro**_
**Change suggestion:**: Git installment link should be moved immediately after the 2 introduction paragraphs
**Why? ** Users may group that line along together with the information about the languages. Users will naturally skim or skip over that paragraph if it doesn't apply to them.
**Personal Experience:** That's exactly what I did when I went through it. Since I knew I didn't need to change languages, I just moved to the next section and missed the part about installing git.

_**Make necessary changes and commit those changes**_
**Change suggestion:** Include information about being prompted to sign in.
**Why?: **When going through that section, the user may be prompted to sign in. If the user doesn't successfully sign in, the user will have issues with later parts.
**Personal Experience:** I incorrectly entered my login information, and I just moved on to the next instructions. This led to me having issues with submitting my changes for review as the "Compare & pull request button" doesn't appear.

**Overall thoughts**
Other than those two issues I had, the tutorial was very simple and well-made! I'm currently a 2nd year CS major wanting to get more involved rather than simply just absorbing lecture material. This led me to a reddit comment mentioning [firsttimersonly.com](https://www.firsttimersonly.com/) with your tutorial as one of the first things listed. I went through a few open source github projects listed on a site also found on that page, though I'm concerned that I can actually contribute or be of use to the projects given my limited knowledge. I'll be sure to checkout that link you provided!

Roshanjossey commented:
Hi @jeffreytram, thank you for your detailed response.

I understand that each user coming here is different and I like to cater to most of the users. At the same time we strive to make this short and simple so that it can be completed within minutes.

That's why I'm taking a path not to make the changes you mentioned. If you still feel strongly about those changes, please open an issue. We'll discuss and make a decision.

I’d urge you to continue contributing to other open source projects.
We also have a slack team to help/mentor people trying to contribute to open source projects.
You can join the team by going to https://goo.gl/PM1Jdh
